src/GraphQL/Execution/InvalidValueException.cs

```diff
@@ -10,5 +10,12 @@ public class InvalidValueException : ExecutionError
         {
             Code = "INVALID_VALUE";
         }
+
+        public InvalidValueException(string variableName, string message, Exception innerException) :
+            base($"Variable '${variableName}' is invalid. {message}", innerException)
+        {
+            Code = "INVALID_VALUE";
+            // note: Codes will also return codes of inner exceptions, plus the Data in the inner exception
```

Comment:
Codes will also return Data in the inner exception ? 😕

Reply:
lol - bad choice of words there - I'll fix it!

Reply:
```suggestion
            // note: this ExecutionError will also return Codes of inner exceptions, plus the Data in the inner exception
```

Reply:
Is that clear?

Reply:
Yes